sdk/python/tests/test_repo_operations.py

```diff
@@ -138,3 +149,32 @@ def test_feastignore_with_stars2():
                 (repo_root / "foo1/c.py").resolve(),
             ]
         )
+
+
+@pytest.mark.parametrize("dataframe_source", [lazy_fixture("simple_dataset_1")])
+@pytest.mark.parametrize("dataframe_source_2", [lazy_fixture("simple_dataset_2")])
```

Comment:
Is there a reason to use `parametrize` here?

```
import pytest


class Fruit:
    def __init__(self, name):
        self.name = name

    def __eq__(self, other):
        return self.name == other.name


@pytest.fixture
def my_fruit():
    return Fruit("apple")


@pytest.fixture
def fruit_basket(my_fruit):
    return [Fruit("banana"), my_fruit]


def test_my_fruit_in_basket(my_fruit, fruit_basket):
    assert my_fruit in fruit_basket
```

Reply:
nope, reason is my ignorance on how to properly pass in fixtures w/o parametrize